core/domain/feedback_services.py

```diff
@@ -302,6 +302,48 @@ def get_message(thread_id, message_id):
         feedback_models.GeneralFeedbackMessageModel.get(thread_id, message_id))
 
 
+# TODO(anubhavsinha98): Remove this function once the message field in all
+# the threads are populated.
+def get_message_count(thread_id):
```

Comment:
Looking at this again, it's fine to inline this in the job (since that's the only place it's being used). That way it will automatically be removed once the job is deleted, and you don't have to rely on the TODO.

In general try to find ways to avoid TODOs since they can end up hanging around for quite a while (you can see there are a number of TODOs in the codebase already :P) Better to not introduce them in the first place.

Reply:
Why not we should make these TODO functions in the test class of one off job, such that if the job is deleted it will also get deleted?

Reply:
I have made all the logics in test file but left this logic in feedback_services as this was being used by the job.

Reply:
Removed this TODO